scripts/exregional_nexus_emission.sh

```diff
@@ -345,6 +350,15 @@ if [ $err -ne 0 ]; then
     err_exit "${message_txt}"
   fi
 fi
+#
+#-----------------------------------------------------------------------
+#
+# Move NEXUS output to INPUT_DATA directory.
+#
+#-----------------------------------------------------------------------
+#
+#mv ${DATA}/NEXUS_Expt_split.nc ${INPUT_DATA}/${NET}.${cycle}${dot_ensmem}.NEXUS_Expt_split.${nspt}.nc
+
```

Comment:
@JianpingHuang-NOAA, if this line is not necessary any more, please remove this whole block (Ln 353-360).

Reply:
Done